apps/omg_child_chain/lib/omg_child_chain/datadog_event/contract_event_consumer.ex

```diff
@@ -76,8 +76,8 @@ defmodule OMG.ChildChain.DatadogEvent.ContractEventConsumer do
   end
 
   # Listens to events via OMG BUS and send them off
-  def handle_info({:internal_event_bus, :data, [data]}, state) do
-    %{event_signature: event_signature} = data
+  def handle_info({:internal_event_bus, :data, data}, state) do
+    %{event_signature: event_signature} = hd(data)
```

Comment:
Do we also need to publish 2nd, 3rd events and so on?

Reply:
Not sure I understand what you mean... we could fetch events coming from a range of blocks all would be the same type. so one type of event per message on the omg bus.

Reply:
I think this :point_up: comment should be placed in the code - otherwise is not clear why we just take the 1st of `data` and ignore the rest

Reply:
Also I spotted the related comment in [similar handler in Watcher](https://github.com/omisego/elixir-omg/pull/1351/files#diff-89f1e052b66822627d1f4f29970083d0R68) - which I believe should apear here as well